### DISCUSSION TITLE
Publish 서버의 Storage데이터를 날리는 방법이 있나요?

### DISCUSSION THREAD

**Discussion by Cha-SeongHun** [naverz/zepeto-studio-kor#1231] [5. Q&A]
테스트하면서 사용하던 Storage 데이터를 Publish 환경에서 사용하지 않도록 기존 데이터를 제거하고 싶습니다.
QR코드 환경은 로컬서버를 재시작하면 Storage가 초기화되지만
Publish환경에서의 Storage데이터는 남아있는 것 같네요.

Storage를 초기화할 수 있는 방법이 따로 있나요?

**Comment by YSHFM** [comment]
저같은 경우는 indext.ts 내 문자열 상수를 하나 지정한 뒤(ex: "TEST_v0.3" ) 해당 문자열 변수를 스토리지 읽어올 때 추가해주는 식으로 되어 있습니다. 초기화가 필요할 경우 상수 값만 변경해주고 있습니다.
![image](https://user-images.githubusercontent.com/106574541/186799446-d4a19e12-2f07-4c17-a97a-326e9f546189.png)
예시코드입니다.
this.m_strServerStorageVer에 "TEST_v0.3"이라고 저장되어 있다면 변경이 필요할 때 v0.4나 이런걸로 변경해주고 테스트하고 있습니다.
